e2e_example/pubspec.yaml

```diff
@@ -1,16 +1,23 @@
 name: e2e_example
-version: 0.2.0
 
 environment:
   sdk: '>=1.9.1 <2.0.0'
 
 dependencies:
-  build: ^0.7.0
-  build_barback: ^0.1.0
+  build: ^0.9.0
+  build_barback: ^0.2.0
 
 dev_dependencies:
-  build_runner: ^0.3.0
+  build_runner: ^0.3.2
 
 transformers:
 - e2e_example:
     $include: 'web/*.txt'
+
+dependency_overrides:
```

Comment:
Not sure if we actually want these, as it can hide dependency version issues? It does keep us on the latest and greatest though, which also has benefits.